draft-packages/badge/docs/Badge.stories.tsx

```diff
@@ -130,6 +129,9 @@ export const LightBadges = () => (
     </div>
   </>
 )
+
+LightBadges.parameters = { chromatic: { disable: false } }
+
```

Comment:
Just going to make a note that we should update the story name here to `DesignSheetDefault` & `DesignSheetReversed` instead of `LightBadges` & `ReversedBadges` at some point (probably best in the PR to change this to the StoryWrapper format) :)

Reply:
Definitely agree. I actually wrote down all the story titles (as shown on storybook) in the PR description, and you'll see that they actually aren't consistent between components. Something we should look at and figure out moving forward :)

Not in this PR though of course :P